tsconfig.json

```diff
@@ -0,0 +1,47 @@
+{
+  "compilerOptions": {
+    /* Basic Options */
+
+    // https://github.com/microsoft/TypeScript/wiki/Node-Target-Mapping as now we support node14.
+    "lib": ["ES2020"],
+    "module": "commonjs",
+    "target": "ES2020",
```

Comment:
Are we sure we want to target ES2020 as a library? I have no experience it this setting for libraries, so maybe we can find some argument for/against certain values.

Reply:
According to the docs (link added as a comment) this is the support for node14(which i understood is the minimum version we support). 

Actually i'm not sure i understood your comment, what's your concern?